src/pages/indexComponents/banner/banner.js

```diff
@@ -40,14 +40,17 @@ const Banner = () => {
 					</Link>{' '}
 					<Link
 						className={clsx(
-							'button button--primary button--md',
+							'button button--primary button--lg',
 							styles.getStarted
 						)}
 						to='https://github.com/herbsjs'
 					>
 						<GitHub fontSize='small' /> Start
 					</Link>
 				</div>
+				<div className={styles.versionInfo}>
+					<p>Last stable version {version}</p>
```

Comment:
@maikmb, in this case the conditional should not display the entire label.

Show 0.0.0 was kind of bad in my opinion

Reply:
Jhow, I've included a version checker to show an empty message in case of crashes.